doc/RELEASE_NOTES.md

```diff
@@ -0,0 +1,828 @@
+
+
+## Removal of Gratuitous Software Installation
+
+The software required to run a particular benchmark, or a particular tool, is no longer gratuitously installed during the execution of a benchmark, or during tool registration.  If a benchmark or tool requires a certain version of software to be present, those checks will be performed and reported to the user as an error if the requirements are not met.
+
+### Change to check command versions instead of RPM versions for `pbench-fio`, `pbench-linpack`, and `pbench-uperf`
+
+The `pbench-fio`, `pbench-linpack`, and `pbench-uperf` no longer perform version checks against RPMs for the required software to execute.  For both `pbench-fio` and `pbench-perf` the reported version string is used from the benchmark command itself.  For `pbench-linpack`, the expected installation directory name is used.
+
+
+## Enhancements to `pbench-linpack`
+
+New `--clients` argument
+
+New `linpack` driver script, can be executed stand-alone
+
+Special thanks to Lukas Doktor for his work on implementing these changes.
+
+
+## Required Use of `--user` with `pbench-move/copy-results`
+
+In preparation for the forth-coming update to the Pbench Server, where the notion of a user is introduced and all result data tar balls are tracked per-user, the `pbench-move-results` and `pbench-copy-results` commands now require that the `--user` switch be provided.  This will help facilitate migrating data into the new version of the Pbench Server.
+
+
+## Support for the New HTTP PUT Method of Posting Tar Balls
+
+In preparation for the forth-coming update to the Pbench Server, support for sending data to a Pbench Server via an HTTP PUT method has been introduced. The new commands, `pbench-results-move` and `pbench-results-push`, provide that functionality.  A token must be generated for the given user via the new `pbench-generate-token` command.
+
+The new `pbench-results-move` and `pbench-results-push` will not work with currently released versions of the Pbench Server (v0.69).  Please consult with a Pbench Server administrator for when the new version will be available for testing purposes, and/or officially released.
+
+
+## Removal of the Use of SCL (Software Collections Library)
+
+For the Pbench Agent we have removed the use of the Software Collections Library (SCL) in order to provide a Python 3 environment for RHEL 7 & CentOS 7 distributions.  We now rely on the Python 3 provided with RHEL 7.9 and CentOS 7.9.
+
+As such, the minimum supported version of RHEL and CentOS is 7.9.
+
+
+## Support for `pbench-trafficgen` dropped entirely
+
+With the release of v0.71 support for `pbench-trafficgen` has been removed in its entirety. The ability to support the behavior of that benchmark is too difficult given the implementation of `pbench-trafficgen` and `pbench-run-benchmark`'s trafficgen support.
```

Comment:
"The ability ... is too difficult" should probably read: "It's too difficult to support the behavior of that benchmark, given ..."